lib/internal/process/per_thread.js

```diff
@@ -33,27 +34,23 @@ const {
 const {
   errnoException,
   codes: {
-    ERR_ASSERTION,
-    ERR_INVALID_ARG_TYPE,
     ERR_INVALID_ARG_VALUE,
     ERR_OUT_OF_RANGE,
     ERR_UNKNOWN_SIGNAL
   }
 } = require('internal/errors');
 const format = require('internal/util/inspect').format;
 const {
+  isInt32,
   validateArray,
   validateNumber,
   validateObject,
 } = require('internal/validators');
+const assert = require('assert');
```

Comment:
This is not being used anywhere, we should remove it (and remove it from the exports).

Reply:
This is actually used here to deprecate the method:
https://github.com/nodejs/node/blob/96e00e03cf57b21e74dd07b7a502eb99e76a35fa/lib/internal/bootstrap/node.js#L293-L296

Should we just leave it there without any modifications since it's deprecated?